### ISSUE TITLE
Need competitive examples on standard benchmarks

### ISSUE THREAD

**Issue by neubig** [clab/dynet#1284]
Separately from the tutorials, it would be nice have examples of competitive models on standard benchmarks. These models should:

1. Give information about how to download the data
2. Be verified to achieve similar accuracy to research papers using this data with similar architectures, or implementations in other toolkits
3. Ideally also provide pre-trained models so people can use them out of the box

This is a "super-issue", and I'll open several sub-issues that point to this.

**Comment by pmichel31415** [comment]
FYI pytorch uses this https://sphinx-gallery.readthedocs.io/en/latest/ to include the examples in the documentation, might be worth to look at
